src/desktop/public/.well-known/assetlinks.json

```diff
@@ -4,6 +4,6 @@
     "namespace": "android_app",
     "package_name": "net.artsy.app",
     "sha256_cert_fingerprints":
-    ["FA:C6:17:45:DC:09:03:78:6F:B9:ED:E6:2A:96:2B:39:9F:73:48:F0:BB:6F:89:9B:83:32:66:75:91:03:3B:9C"]
+    ["72:70:C3:1C:74:A4:86:D8:F7:31:08:E0:2E:72:D6:91:00:4D:57:35:16:8B:9C:DA:B1:65:5D:49:1D:33:2E:88"]
```

Comment:
@MounirDhahri Never mind.

Reply:
With the current way this file is being served now, we can only serve one fingerprint per host and this is why I chose the release one. Ideally, we should not be serving this file this way, instead, it should be served dynamically from a node server or something similar where we can have full control of it, like we do for iOS here https://github.com/artsy/artsy-eigen-web-association.

On how we can use debug and release versions both, google recommends creating two different variants for releases  as in here https://developer.android.com/training/app-links/verify-site-associations#publish-json but also explicitly mentions not publishing the app with dev/test URLs.

For now, the check for the sha256 will fail on debug builds and we will be getting deep links served instead of app links but that's fine I guess